Tests/Realm.Tests/Sync/SyncTestHelpers.cs

```diff
@@ -310,10 +313,14 @@ public static Task<T> WaitForSessionError<T>(Action<Session> sessionAssertions)
                     tcs.TrySetException(ex);
                 }
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Can we suppress that with a global suppressions attribute rather than pragmas? That way we won't have to hunt all the pragmas when we start fixing things.